components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2280,6 +2280,14 @@ where
             |_| { unreachable!() }
         );
 
+        for new_region in &regions {
+            if new_region.get_id() == derived.get_id() {
+                continue;
+            }
+            ctx.host
+                .on_region_split(derived.get_id(), new_region.get_id());
```

Comment:
It would be better to pass all `regions` to coprocessor in one call. It's possible for a coprocessor that wants all regions' information. Besides, it would cost more function calls, and prevent some optimization such as vectorization.